src/com/owncloud/android/lib/resources/comments/CommentFileRemoteOperation.java

```diff
@@ -0,0 +1,114 @@
+/* ownCloud Android Library is available under MIT license
```

Comment:
shouldn't we rename it from oC to Nc?

Reply:
Correct, for new added files we can do this.